runtime/src/snapshot_utils.rs

```diff
@@ -169,7 +209,8 @@ impl Ord for SlotSnapshotPaths {
     }
 }
 
-pub fn package_snapshot<P: AsRef<Path>, Q: AsRef<Path>>(
+/// Package up slot snapshot files, snapshot storages, and slot deltas for a full snapshot.
+pub fn package_snapshot<P, Q>(
```

Comment:
nit: package_snapshot -> package_full_snapshot

Reply:
Yah, I wasn't sure what to do; either rename all the existing things to `full_`, or not. Since currently a "snapshot" is a full snapshot, that's why I didn't add `full_` to all the functions/variables. Inside all the incremental snapshot functions, that's where everything is either `full_` or `incremental_`.

I also wasn't sure how far to take the renaming. Like should it cause gossip/RPC to change names too? I'm not sure on that part. What do you think @carllin ?

Reply:
Oh, and also the on-disk snapshots right now are also named "snapshot" and are full snapshots. To handle renaming the on-disk snapshots, I'd need to add code to support both `full_snapshot` and `snapshot` files/directories for a while, for backwards compatibility.

Reply:
I think existing identifiers should be prefixed with `full_` to avoid any ambiguity.

and file system and rpc / gossip should remain same.

in other words, `full_` rename should occur in all code where appropriate for humans, while keeping compatibility.

it's a bit daunting for me. but could you do that once for all for us, considering your pr's high quality of plumbing?

Reply:
@ryoqun Yes, will do.

Reply:
OK, done!

Reply:
> in other words, full_ rename should occur in all code where appropriate for humans, while keeping compatibility.

Sorry late to the party here, agreed, especially given we're introducing `incremental_snapshots`